src/osd/osd_types.h

```diff
@@ -4117,7 +4117,7 @@ struct object_info_t {
   explicit object_info_t(bufferlist& bl) {
     decode(bl);
   }
-  object_info_t operator=(bufferlist& bl) {
+  object_info_t &operator=(bufferlist& bl) {
```

Comment:
@tchaikov  May I delete this function directly or append  '= delete' to the function declaration?